metadata-ingestion-modules/dagster-plugin/setup.py

```diff
@@ -22,7 +22,7 @@ def get_long_description():
 
 base_requirements = {
     # Actual dependencies.
-    "dagster >= 1.3.3",
+    "dagster >= 1.3.3,<1.10",
```

Comment:
upper bound dependencies put us in a bad position for the future

is this the error that is blocking us?

```
+ mypy --show-traceback --show-error-codes src/ tests/ examples/
tests/unit/test_dagster.py:147: error: "ExecuteInProcessResult" has no attribute "get_job_success_event"  [attr-defined]
Found 1 error in 1 file (checked 20 source files)
```

1.10.0 was released 12h ago and I haven't seen any mention to the removal of the method `get_job_success_event` 
 so maybe this is an error in the release https://github.com/dagster-io/dagster/releases/tag/1.10.0

I would suggest to skip this specific version for the moment, and cross-fingers they fix in the next hotfix release

```suggestion
    "dagster >= 1.3.3,!=1.10.0",
```

Reply:
Agreed with Sergio. Also, every upper bound should have a comment next to it explaining why it was added in the first place